packages/date-time-utilities/src/dateMath/DateMath.ts

```diff
@@ -0,0 +1,417 @@
+import { DayOfWeek, MonthOfYear, FirstWeekOfYear, DateRangeType } from '../dateValues/DateValues';
+import TimeConstants from '../dateValues/TimeConstants';
+import { assertNever } from '@uifabric/utilities';
```

Comment:
One more suggestion you can take or leave: it looks like the only reason for the dependency on `@uifabric/utilities` is this `assertNever` call (which literally all that does is throw an error). So I think it might be good to get rid of the dependency and just throw an error manually.

Reply:
Sure, makes sense to me :)
Will update the code :)